client/config/crd/snapshot.storage.k8s.io_volumesnapshotcontents.yaml

```diff
@@ -148,7 +148,7 @@ spec:
               sourceVolumeMode:
                 description: |-
                   SourceVolumeMode is the mode of the volume whose snapshot is taken.
-                  Can be either “Filesystem” or “Block”.
+                  Can be either "Filesystem" or "Block".
```

Comment:
This yaml is automatically generated.

Reply:
@xing-yang do you know where is the source of this generated file? it's a Chinese quote char which needs to be fixed since it would fail our yamllint check test.

Reply:
I think it's generated by client/apis/volumesnapshot/v1/types.go, and also fix it

Reply:
seems it's not source file, vendor will fail the test